package-lock.json

```diff
@@ -3,82 +3,132 @@
   "version": "21.0.0",
   "lockfileVersion": 1,
   "requires": true,
-  "dependencies": {
-    "@aashutoshrathi/word-wrap": {
+  "packages": {
```

Comment:
Hey, @VasylBiliak, in your task, you SHOULD work only in YOUR folder.
So, please remove the changes to the files .gitignore and eslintrc.js from your pull request. That is mean, you have to revert the files to their state in the main branch:

https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/.gitignore
https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/.gitignore